src/app/zap-templates/zcl/data-model/chip/mode-base-cluster.xml

```diff
@@ -22,12 +22,14 @@ Due to the ModeBase cluster not having a clustre ID, these enems will need to be
 These enems can be used once the zap generation tool supports aliased and derived clusters.
 -->
 <!--  <enum name="StatusCode" type="ENUM8">-->
+<!--    <cluster code="0xf123"/>-->
```

Comment:
Those f123 looks a bit weird.... of you mean "Mode Base Cluster ID", maybe write that?  Because f123 looks like a real cluster id.

Reply:
Those where temporary while I was trying to generate some code. I have since replaced them with `0x...`.